### ISSUE TITLE
Reduce noise effect on ocean floor

### ISSUE THREAD

**Issue by ftomassetti** [Mindwerks/worldengine#101]
Let's check if we can do that in a smooth way (no "jumps" between ocean and land)

**Comment by tibor95** [comment]
Do you mean to make beaches&coasts less steep? Because "ocean floor" is something different...

**Comment by ftomassetti** [comment]
no, I am thinking about the middle of the ocean where there are sometimes the signs of perlin noise which are too evident. So I was thinking to reduce the noise for each cell that is part of the ocean, but we should avoid discontinuity between ocean and land. We could also think to apply antialias to the ocean floor

**Comment by tibor95** [comment]
I see, thanks for explanation

**Comment by psi29a** [comment]
@tibor95 that is another issue that will be addressed.  I'm thinking there could be a filter that allows for a less steep beach/coast that flows into the sea for a few km then drops off, like the continental shelf. That won't happen everywhere of course because there things like the Cliffs of Dover. ;)

**Comment by ftomassetti** [comment]
Or the cliffs of moher (https://d1u1p2xjjiahg3.cloudfront.net/ec358c37-6bf1-4278-9700-993f1daa451c.jpg) you have to considered that I used to live in Ireland :D

**Comment by tibor95** [comment]
I am not a traveler, but I was there!!! :)

For my purposes I need flat beaches (at least somewhere) because ports needs relatiely big flat space only small bit above a sea level

**Comment by ftomassetti** [comment]
you are right: Cliffs are nice somewhere but they should not be the norm. We have never realized the coasts were so steep so far, maybe we should consider aggressive sea erosion on coasts. Or increase the amount of land moved by erosion: a food portion of the North of Italy was originated from debris arriving from the Alps and transported by rivers: this kind of land ends in the sea very smoothly

**Comment by psi29a** [comment]
Since this issue has a lot of talk about smoother coastlines, can we rename the topic. ;)

Start another issue to tackle the ocean floor noise?

**Comment by psi29a** [comment]
Duplicate?: https://github.com/Mindwerks/worldengine/issues/60

**Comment by ftomassetti** [comment]
Renaming (the original issue is indeed discussed in #60)
